Source/WebKit/Shared/OpaqueTransports.serialization.in

```diff
@@ -0,0 +1,246 @@
+[Legacy] alias EncryptedMediaOpaqueData = std::span<const uint8_t>;
+[Legacy] alias WebSocketChannelOpaqueData = std::span<const uint8_t>;
+[Legacy] alias WebTransportSessionOpaqueData = std::span<const uint8_t>;
+[Legacy] alias LibWebRTCNetworkOpaqueData = std::span<const uint8_t>;
+[Legacy] alias RTCOpaqueData = std::span<const uint8_t>;
+[Legacy] alias BindAccessibilityOpaqueData = std::span<const uint8_t>;
+[Legacy] alias InjectedBundleOpaqueData = std::span<const uint8_t>;
+[Legacy] alias WebPageLoadOpaqueData = std::span<const uint8_t>;
+[Legacy] alias GPUProcessBookmarkOpaqueData = std::span<const uint8_t>;
+[Legacy] alias LibWebRTCCodecsProxyFormatDescriptionOpaqueData = std::span<const uint8_t>;
+[Legacy] alias LibWebRTCCodecsProxyDecodeFrameOpaqueData = std::span<const uint8_t>;
+[Legacy] alias RemoteGraphicsContextGLOpaqueData = std::span<const uint8_t>;
+[ProvenUnstructuredData='Pixel buffer data'] alias RemoteGraphicsContextGLPixelOpaqueData = std::span<const uint8_t>;
+
+#if PLATFORM(IOS_FAMILY)
+[Legacy] alias WebPageIconOpaqueDate = std::span<const uint8_t>;
+#endif
+
+#if PLATFORM(COCOA)
+[Legacy] alias WebPageAccessibilityOpaqueData = std::span<const uint8_t>;
+#endif
+
+#if PLATFORM(COCOA)
+[Legacy] alias WebPagePasteboardOpaqueData = std::span<const uint8_t>;
+#endif
+
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
+[Legacy] alias WebPageHEICOpaqueData = std::span<const uint8_t>;
+#endif
+
+#if ENABLE(IMAGE_ANALYSIS_ENHANCEMENTS)
+[Legacy] alias WebPageReplaceImageOpaqueData = std::span<const uint8_t>;
+#endif
+
+#if ENABLE(PDF_PLUGIN) && PLATFORM(MAC)
+[Legacy] alias WebPageOpaquePDFData = std::span<const uint8_t>;
+#endif
+
+#if !HAVE(WK_SECURE_CODING_SECTRUST)
+[Legacy] alias CoreIPCSecTrustOpaqueData = std::span<const uint8_t>;
+#endif
+
+[Legacy] alias CoreIPCDataOpaqueData = std::span<const uint8_t>;
+[Legacy] alias StateObjectOpaqueData = Vector<uint8_t>;
+
+[ProvenUnstructuredData='Pixel buffer data'] alias ByteArrayPixelBufferOpaqueData = std::span<const uint8_t>;
+[ProvenUnstructuredData='Pixel buffer data'] alias PixelBufferSourceViewOpaqueData = std::span<const uint8_t>;
+
+#if ENABLE(SANDBOX_EXTENSIONS)
+[Legacy] alias SandboxExtensionImpOpaqueData = std::span<const uint8_t>;
+#endif
+
+[Legacy] alias PushEvenOpaqueData = std::span<const uint8_t>;
+
+[Legacy] alias NetworkConnectionLoadOpaqueData =  Vector<uint8_t>;
+[Legacy] alias NetworkConnectionServerKeyOpaqueData = Vector<uint8_t>;
+[Legacy] alias ServiceWorkerServerKeyOpaqueData = Vector<uint8_t>;
+[Legacy] alias WebPageProxyURLSchemeTaskOpaqueData = Vector<uint8_t>;
+[Legacy] alias WebPageProxyBindRemoteDataTokenOpaqueData = Vector<uint8_t>;
+[Legacy] alias WebPageBindRemoteDataTokenOpaqueData = Vector<uint8_t>;
+[Legacy] alias WebPageImageOpaqueData = Vector<uint8_t>;
+[Legacy] alias GPURemoteQueueOpaqueData = Vector<uint8_t>;
+[Legacy] alias GPURemoteQueueTextureOpaqueData = Vector<uint8_t>;
+[Legacy] alias GPURemoteBufferOpaqueData = Vector<uint8_t>;
```

Comment:
`[Legacy]` seems like the wrong term here.

We will continue to need to pass, in current and upcoming web specifications, `uint8_t` data across IPC

Reply:
Yes `RequiredBySpec=` sounds good :) It just came to mind since the [PR](https://github.com/WebKit/WebKit/pull/50706) I am looking to merge adds 2 new Vector<uint8_t> instances being passed across IPC:
```
+    Vector<uint8_t> buffer;
+    Vector<KeyValuePair<int32_t, uint64_t>> renderFlags;
+    Vector<uint8_t> indices;
```

For the open PR the spec link is:
https://immersive-web.github.io/model-element/

For the `GPURemote*` ones in this PR the spec link is https://github.com/gpuweb/gpuweb